### PR TITLE
Adjust backup-and-maintenance.sh for locationlog_history table

### LINKED ISSUE #5076: locationlog1to1: backup-and-maintenance.sh to adjust
**Describe the bug**
With #4995:
1. We keep only one entry per node in `locationlog` but pfmon `locationlog_cleanup` job will remove this entry after 1 week by default. I'm not sure it's a problem.

1. `backup-and-maintenance.sh` script has several references to `locationlog_archive` table which didn't exist anymore:
- https://github.com/inverse-inc/packetfence/blob/15e5365172113258c7206bc7d492cac11e15d9af/addons/backup-and-maintenance.sh#L116
- https://github.com/inverse-inc/packetfence/blob/15e5365172113258c7206bc7d492cac11e15d9af/addons/backup-and-maintenance.sh#L162

These references need to be replace by `locationlog_history` I guess.

### PR DESCRIPTION
**Pull request by nqb**
# Description
- Exclude `locationlog_history` from `mysqldump`
- Remove call to `database-cleaner.pl` because `pfmon` already do the job
- Update docs

# Impacts
- `backup-and-maintenance.sh` script

# Issue
fixes #5076 

# Delete branch after merge
YES

# NEWS file entries
## Bug Fixes
* Adjust backup-and-maintenance.sh for `locationlog_history` table (#5076)

### CODE REVIEW

File: addons/backup-and-maintenance.sh [hunk 1]
```diff
@@ -113,7 +109,7 @@ should_backup(){
 }
 
 backup_db(){
-    /usr/local/pf/addons/database-cleaner.pl --table=locationlog_archive --date-field=end_time --older-than="1 MONTH"
+    /usr/local/pf/addons/database-cleaner.pl --table=locationlog_history --date-field=end_time --older-than="1 MONTH"
```
Comment (julsemaan) [inline addons/backup-and-maintenance.sh]: This is already cleaned by the locationlog_cleanup job so there is no point on even keeping that line

Comment (nqb) [reply addons/backup-and-maintenance.sh]: Done

File: docs/PacketFence_Installation_Guide.asciidoc [hunk 2]
```diff
@@ -4307,7 +4307,7 @@ Keeping Tables Small
 
 Over time, some of the tables will grow large and this will drag down performance (this is especially true on a wireless setup).
 
-One such table is the `locationlog` table. We recommend that closed entries in this table be moved to the archive table `locationlog_archive` after some time. A closed record is one where the `end_time` field is set to a date (strictly speaking it is when `end_time` is not null and not equals to 0).
+One such table is the `locationlog` table. We recommend that closed entries in this table be moved to the archive table `locationlog_history` after some time. A closed record is one where the `end_time` field is set to a date (strictly speaking it is when `end_time` is not null and not equals to 0).
```
Comment (julsemaan) [inline docs/PacketFence_Installation_Guide.asciidoc]: This line isn't true anymore, this happens auto-magically so we can just remove this

Comment (nqb) [reply docs/PacketFence_Installation_Guide.asciidoc]: You're right, I missed it. I will adjust.

**Comment by nqb** [comment]
@julsemaan, it should be good now.

File: addons/backup-and-maintenance.sh [hunk 3]
```diff
@@ -1,19 +1,16 @@
 #!/bin/bash
 #
-# Database maintenance and backup
+# Backup of PF tree and database
```
Comment (julsemaan) [inline addons/backup-and-maintenance.sh]: PF tree ?

**Pull request merged**